app/src/main/res/values/themes.xml

```diff
@@ -36,9 +36,7 @@
     </style>
 
     <style name="DialogTheme" parent="Theme.AppCompat.Light.Dialog">
-        <!-- Avoid duplicate background beneath the dialog -->
         <item name="colorAccent">@color/theme</item>
-        <item name="android:windowBackground">@android:color/transparent</item>
```

Comment:
I can't remember why I add this 🤔 
But it looks to work well without this according to your screenshots.